components/shindig-wso2-features/src/main/javascript/wso2features/remoteClient/feature.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<!--
+  ~ Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<feature>
+    <name>wso2-gadgets-remoteClient</name>
+    <dependency>wso2-gadgets-core</dependency>
```

Comment:
Are you using any methods defined in wso2-gadgets-core feature? If not remove this dependency.